crates/re_data_ui/src/editors.rs

```diff
@@ -0,0 +1,221 @@
+// TODO(jleibs): Turn this into a trait
+
+use egui::NumExt as _;
+use re_data_store::{DataStore, LatestAtQuery};
+use re_log_types::EntityPath;
+use re_query::ComponentWithInstances;
+use re_types::{
+    components::{Color, Radius, ScalarScattering, Text},
+    Component, Loggable,
+};
+use re_viewer_context::{UiVerbosity, ViewerContext};
+
+#[allow(clippy::too_many_arguments)]
+fn edit_color_ui(
+    ctx: &ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    _verbosity: UiVerbosity,
```

Comment:
I think it will eventually. I suspect there will be both a Small/Inline editor for some things, when it's appropriate, but I could imagine a way of expanding that to a more "full" multiline editor that we won't always want to show in the selection panel by default.